doc/userland-tools.md

```diff
@@ -0,0 +1,47 @@
+# userland-tools
+
+oi-userland comes with some tools used all over the build system and 
+these tools reside in tools/ directory.
+
+## userland-zone
+
+_userland-zone_ is a tool to manage a lifecycle of build zones in oi-userland. 
+The intended and main use case for it are in continuous integration system and do build in a clean environment.
```

Comment:
in our (the) continuous integration system and provides a clean build environment.